docs/content/policy-testing.md

```diff
@@ -118,11 +119,12 @@ FAIL: 1/4
 ## Test Format
 
 Tests are expressed as standard Rego rules with a convention that the rule
-name is prefixed with `test_`.
+name is prefixed with `test_`. It is best practice for tests to be placed in a package suffixed with `_test`. 
```

Comment:
While I agree, I think we can be a little more lenient in the OPA docs. 

I'd replace:
> It is best practice for tests to be placed in a package suffixed with `_test`.

with
> It's a good practice for tests to be placed in a package suffixed with `_test`, but not a requirement.